core/src/com/unciv/models/ruleset/unique/Unique.kt

```diff
@@ -197,6 +197,9 @@ class Unique(val text: String, val sourceObjectType: UniqueTarget? = null, val s
                 state.ourCombatant != null && state.ourCombatant.getHealth() > condition.params[0].toInt()
             UniqueType.ConditionalBelowHP ->
                 state.ourCombatant != null && state.ourCombatant.getHealth() < condition.params[0].toInt()
+            UniqueType.ConditionalHasNotUsedOtherActions ->
+                state.unit != null &&
+                state.unit.run { religiousActionsUnitCanDo().all { abilityUsesLeft[it] == maxAbilityUses[it] } }
```

Comment:
Nope, that one does not touch UniqueTypes or conditionals. Here I'm killing the specialized "Can construct [Holy site] if it hasn't used other actions yet" because "Can construct [Holy site]" exists anyway and with a conditional becomes cleaner: "Can construct [Holy site] <if it hasn't used other actions yet>"